src/test/java/teammates/ui/webapi/CreateAccountRequestActionTest.java

```diff
@@ -13,6 +14,7 @@
 /**
  * SUT: {@link CreateAccountRequestAction}.
  */
+@Ignore
```

Comment:
Let's not use `@Ignore` tag, you can use @Test(enable = false) (similar to other PRs)

Reply:
changed to the `@test` annotation